LongWarOfTheChosen/Src/LW_Overhaul/Classes/LWTacticalMissionUnitSpawner.uc

```diff
@@ -477,6 +480,8 @@ static function LoadLiaisonFromOutpost(XComGameState_LWOutpost Outpost,
 		{
 			ItemState = XComGameState_Item(NewGameState.CreateStateObject(class'XComGameState_Item', ItemReference.ObjectID));
```

Comment:
Would you mind changing this to `ModifyStateObject()` and removing the call to `AddStateObject()`? I'd like to update to the new WOTC API a we go.

Reply:
ok will do, did not catch that is how `SeqAct_SpawnUnitFromAvenger` does it. Will make the fix.

Reply:
done